sklearn/metrics/tests/test_ranking.py

```diff
@@ -696,11 +696,14 @@ def test_binary_clf_curve_implicit_pos_label(curve_func):
 
     # The error message is slightly different for bytes-encoded
     # class labels, but otherwise the behavior is the same:
-    msg = ("y_true takes value in {b'a', b'b'} and pos_label is "
-           "not specified: either make y_true take "
-           "value in {0, 1} or {-1, 1} or pass pos_label "
-           "explicitly.")
-    with pytest.raises(ValueError, match=msg):
+    msg = (("y_true takes value in {b'a', b'b'} and pos_label is "
```

Comment:
I thought a lot about it, but ultimately it seemed like the best way was to check for both the new error message and the old one together.

Reply:
I think that we can pass the error message in the parameterization linked to the metric tested.
If needed, we can isolate the test just for this usecase.